### PR TITLE
docs: add README.md for cli tool

### PR DESCRIPTION
**Pull request by SLdragon**

### CODE REVIEW

File: packages/api2teams/src/resources/project-template/README.md [hunk 1]
```diff
@@ -17,19 +38,99 @@ In the src folder, there are 4 sub folders:
 * commands folder
     - This folder contains bot command handlers, when user send messages, it will match the api, and send back request adaptive card. There is a special command handler which named `helpCommandHandler` which is used to show help messages.
 
+* index.ts file
+    - Index file using `restify` as web service framework to host bot server, and register command handlers and action handlers for bot to handle use messages and button clicking event. 
+
 ![](./images/project-structure.png)
```
Comment (MuyangAmigo) [inline packages/api2teams/src/resources/project-template/README.md:45]: Can we exclude the images from user disk

Comment (SLdragon) [reply packages/api2teams/src/resources/project-template/README.md:45]: sure

File: packages/api2teams/src/resources/project-template/README.md [hunk 2]
```diff
@@ -17,19 +38,99 @@ In the src folder, there are 4 sub folders:
...
 
 # Workflow
-![](./images/workflow-command-workflow.png)
-- User input command message to the bot, for example `GET /pets`
+For example, below is the API to get pets list:
```
[the remark this answers is not in the document]
Comment (SLdragon) [reply packages/api2teams/src/resources/project-template/README.md:47]: This part is to explain the workflow of the generated teams app, if we don't want this part, we can put it into wiki page

File: packages/api2teams/src/resources/project-template/README.md [hunk 3]
```diff
@@ -1,9 +1,30 @@
...
+# Quick Start
+1. First, select the Teams Toolkit icon on the left in the VS Code toolbar.
+1. In the Account section, sign in with your [Microsoft 365 account](https://docs.microsoft.com/microsoftteams/platform/toolkit/accounts) if you haven't already.
+1. Press F5 to start debugging which launches your app in Teams using a web browser. Select `Debug (Edge)` or `Debug (Chrome)`.
+1. When Teams launches in the browser, select the Add button in the dialog to install your app to Teams.
+1. Click message input dialog, you can see supported commands.
+1. Input command in the chat and send it to bot, and then you can get adaptive card
+
+# Converted Teams App Folder Structure
+The generated Teams APP is a normal Teams Toolkit project. In the src folder, there are 4 subfolders and a index.ts file insdie:
```
[the remark this answers is not in the document]
Comment (SLdragon) [reply packages/api2teams/src/resources/project-template/README.md:26]: sure

File: packages/api2teams/src/resources/project-template/README.md [hunk 4]
```diff
@@ -17,19 +38,99 @@ In the src folder, there are 4 sub folders:
...
+![](./images/flow2.png)
+
+- User input the command message `GET /pets?limit=3`
+- Bot service receive this message, and trigger `listPetsCommandHandler`
+- Due to the required parameter `limit` is inside the user message, so `listPetsCommandHandler` would call API service and use the response data to render `PetsListCard.json` card.  
+-  Then `listPetsCommandHandler` would send back `PetsListCard.json` adaptive card to user.
+  ![](./images/workflow3.png)
+
+# Customize adaptive cards
+The generated adaptive card can be found in `src/adaptiveCards` folder and you can customize the cards according to your own personal needs. You can use [adaptive card designer](https://adaptivecards.io/designer) from browser to design your cards.
```
[the remark this answers is not in the document]
Comment (SLdragon) [reply packages/api2teams/src/resources/project-template/README.md:116]: Can you give some links in other template? I didn't find it

File: packages/api2teams/src/resources/project-template/README.md [hunk 5]
```diff
@@ -17,19 +38,99 @@ In the src folder, there are 4 sub folders:
...
+
+# Learn and customize command bot project
+The generated project is a general TeamsFx command response bot project, and you can custimize and add your own command handlers, for more information, you can refer [this wiki page](https://github.com/OfficeDev/TeamsFx/wiki/Respond-to-chat-commands-in-Teams#how-command-and-response-works)
```
[the remark this answers is not in the document]
Comment (SLdragon) [reply packages/api2teams/src/resources/project-template/README.md:119]: Yes, already point to the wiki page
```
you can refer [this wiki page](https://github.com/OfficeDev/TeamsFx/wiki/Respond-to-chat-commands-in-Teams#how-command-and-response-works)
```

**Pull request merged**